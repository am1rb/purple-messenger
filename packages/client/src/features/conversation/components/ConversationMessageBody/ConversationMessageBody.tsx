import React from "react";

export interface ConversationMessageBodyProps {
  children: React.ReactNode;
}

function ConversationMessageBody({ children }: ConversationMessageBodyProps) {
  return <>{children}</>;
}

export default ConversationMessageBody;
