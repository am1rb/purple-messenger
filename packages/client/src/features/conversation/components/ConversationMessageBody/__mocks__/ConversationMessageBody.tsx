import React from "react";

export interface ConversationMessageBodyProps {
  children: React.ReactNode;
}

function ConversationMessageBody({ children }: ConversationMessageBodyProps) {
  return <span data-testid="mock-conversation-message-body">{children}</span>;
}

export default ConversationMessageBody;
