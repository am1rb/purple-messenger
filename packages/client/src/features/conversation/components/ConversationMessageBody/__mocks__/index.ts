export { default } from "./ConversationMessageBody";
export * from "./ConversationMessageBody";
