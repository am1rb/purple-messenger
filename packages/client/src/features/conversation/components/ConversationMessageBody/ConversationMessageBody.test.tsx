import React from "react";
import { render } from "@testing-library/react";
import ConversationMessageBody, {
  ConversationMessageBodyProps,
} from "./ConversationMessageBody";

const sharedProps: ConversationMessageBodyProps = {
  children: "the message body",
};

describe("The <ConversationMessageBody /> tests", () => {
  it("Should match the snapshot", () => {
    const { container } = render(<ConversationMessageBody {...sharedProps} />);
    expect(container).toMatchSnapshot();
  });
});
