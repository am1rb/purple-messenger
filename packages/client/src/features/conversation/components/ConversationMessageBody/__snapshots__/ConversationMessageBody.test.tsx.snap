// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The <ConversationMessageBody /> tests Should match the snapshot 1`] = `
<div>
  the message body
</div>
`;
